src/ModernHttpClient/iOS/NSUrlSessionHandler.cs

```diff
@@ -46,7 +46,7 @@ public class NativeMessageHandler : HttpClientHandler
         public NativeMessageHandler(): this(false, false) { }
         public NativeMessageHandler(bool throwOnCaptiveNetwork, bool customSSLVerification)
         {
-            var config = NSUrlSessionConfiguration.EphemeralSessionConfiguration;
+            var config = NSUrlSessionConfiguration.DefaultSessionConfiguration;
```

Comment:
From the doc, default mode also sticks the credential into the keychain. We need to understand the search parameter settings it uses when adding the creds into the keychain. We *may* need to change the way we store the password in the keychain so it exactly matches this pattern. Otherwise, we won't be able to handle password change scenarios.

If you can determine the keychain insert parameters, Azim is the right person to mod our keychain behavior.